app/views/finders/_facet_collection.html.erb

```diff
@@ -1,4 +1,11 @@
 <div class="filter-form">
+  <% if content_item.all_content_finder? %>
+      <%= render partial: 'filter_button'%>
+    <% elsif !content_item.all_content_finder? %>
+      <% if facets.select(&:filterable?).any? %>
+        <%= render partial: 'filter_button'%>
+      <% end %>
+  <% end %>
```

Comment:
Nitpick: there still seems to be something wrong with the indentation here.

Reply:
Whoops, my bad!

Reply:
Ummm... still not quite right. The inner lines are partly indented by 2 spaces, partly by 4. Can you take another look?

Reply:
@andysellick My bad, I've been using the `tab` key on Visual Studio Code to indent this, so I'm guessing it's judgement is wrong with `.erb` files.

Reply:
Actually it's probably my fault. @andysellick Does it look OK now? If I got it wrong again, may be worth writing out the correct indentation 😅